### PR TITLE
EN6-183 WidgetConfiguration Format

### PR DESCRIPTION
**Pull request by ffleandro**
Also fixes EN6-182;

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/widgettype/validators/ContentListViewerWidgetValidator.java [hunk 1]
```diff
@@ -119,7 +119,6 @@ protected void validateFilters(WidgetConfigurationRequest widget, BeanPropertyBi
                 && type.hasParameter(WIDGET_CONFIG_KEY_MAXELEMFORITEM)
                 && type.hasParameter(WIDGET_CONFIG_KEY_MAXELEMENTS)
                 && StringUtils.isNotEmpty((String) config.get(WIDGET_CONFIG_KEY_CONTENTTYPE))
-                && StringUtils.isEmpty((String) config.get(WIDGET_CONFIG_KEY_CATEGORIES))
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/widgettype/validators/ContentListViewerWidgetValidator.java]: Sure about this?

**Comment by joewhite101** [comment]
@ffleandro double check the tests on this one. Looks like possibly a legit failure. And double check the deletion in the validator. Could be correct but want to be sure.

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/widgettype/validators/ContentListViewerWidgetValidator.java (hunk 1 above)
Comment (ffleandro) [reply plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/widgettype/validators/ContentListViewerWidgetValidator.java]: Yeah, it was throwing CastException with the new object format.
Didn't see a need to keep that kind of validation with a list.

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well
 No test results found.
All is Well

**Pull request merged**